src/contributing/index.md

```diff
@@ -2,13 +2,22 @@
 
 There's more to running a distribution than just writing code.
 
-To contribute to the Void packages repository, start by reading the
+If you find an issue with a package and you manage to fix it, you can add your
+solution to the official package repository. New packages and updates are also
+welcome! To contribute to the Void packages repository, start by reading the
 [CONTRIBUTING](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md)
 document in the void-packages GitHub repository.
 
-To contribute to this Handbook, read
-[CONTRIBUTING](https://github.com/void-linux/void-docs/blob/master/CONTRIBUTING.md)
-in the void-docs repository.
+If you are unable to fix a package yourself, make sure to [open an
+issue](https://github.com/void-linux/void-packages/issues/new) in the
+void-packages repository. This makes maintainers aware of the issue, allows
+others to fix it, and can work as a place for anyone to share workarounds in the
+meantime. Remember to check the existing issues for your problem! There are also
+some [things you can do to improve your bug report](./debug.md).
```

Comment:
improve your bug report could be changed to something along the lines of assist us in finding the probelm

Reply:
Done.